.buildkite/pipeline.yml

```diff
@@ -1,6 +1,36 @@
 steps:
-  - commands:
+  - label: ":hammer: running tests"
+    commands:
       - "make ci"
-    name: ":hammer: build"
+        #- "make test" #currently no ct defined, causes build error
+    key: "tests"
+    agents:
+      queue: "erlang"
+
+  - wait
```

Comment:
You don't need the wait if you have a depends_on attribute

Reply:
good catch